Dockerfile-build

```diff
@@ -175,12 +175,8 @@ RUN bash -c 'if [ `arch` = "ppc64le" ]; then \
 RUN pip install numpy==1.16.4 scipy==1.2.1 setuptools==39.0.1
```

Comment:
These are here only because they need to be done super early to avoid issue when doing full requirements file handling by pip.  But nominally, let's do everything in requirements except for these basic things.